js/donations-visualiser.js

```diff
@@ -125,6 +125,27 @@ d3.select("#receipt-type-select-all").on("click", function() { selectAll('#recei
 d3.select("#receipt-type-select-clear").on("click", function() { clearSelection('#receipt_type_select'); });
 d3.select("#clear-search").on("click", clearSearch);
 
+// on 'escape' key press, close the info window, and zoomToFit
+window.addEventListener("keydown", function (event) {
+  if (event.defaultPrevented) return;
+
+  var handled = false;
```

Comment:
handled seems like a funny name for this. it basically reflects whether the key was esc right?